podcasts/End of Year/Stories/Views/PodcastCover.swift

```diff
@@ -49,16 +49,28 @@ struct PodcastCover: View {
             ImageView(image: image)
                 .cornerRadius(big ? 8 : 4)
 
-            Action {
-                image = nil
-                KingfisherManager.shared.retrieveImage(with: ServerHelper.imageUrl(podcastUuid: podcastUuid, size: 280)) { result in
-                    switch result {
-                    case .success(let result):
-                        image = result.image
-                    default:
-                        break
+                .onAppear {
```

Comment:
We need to use `onAppear` so images are loaded when we render the view to snapshot. 🤷